plugin/pkg/admission/exec/admission.go

```diff
@@ -77,17 +86,20 @@ func NewDenyEscalatingExec() *DenyExec {
 	}
 }
 
-// NewDenyExecOnPrivileged creates a new admission controller that is only checking the privileged
-// option. This is for legacy support of the DenyExecOnPrivileged admission controller.
-// Most of the time NewDenyEscalatingExec should be preferred.
-func NewDenyExecOnPrivileged() *DenyExec {
-	return &DenyExec{
-		Handler:     admission.NewHandler(admission.Connect),
-		hostNetwork: false,
-		hostIPC:     false,
-		hostPID:     false,
-		privileged:  true,
+var _ admission.ValidationInterface = &DenyExec{}
+var _ = genericadmissioninitializer.WantsExternalKubeClientSet(&DenyExec{})
+
+// SetInternalKubeClientSet implements the WantsInternalKubeClientSet interface.
+func (d *DenyExec) SetExternalKubeClientSet(client kubernetes.Interface) {
```

Comment:
the functions are moved up to be closer to type, as requested in other relevant PRs.